### PR TITLE
Keyboard option in VGSExpDateConfiguration.

### PR DESCRIPTION
**Pull request by dmytrokhl**
## Description of changes
Add keyboard option in VGSExpDateConfiguration

## Testing
UI Tests

### CODE REVIEW

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSExpDateTextField.swift [hunk 1]
```diff
@@ -132,10 +123,35 @@ extension VGSExpDateTextField: UIPickerViewDelegate, UIPickerViewDataSource {
...
+    textField.inputView = picker
+    updateYearsDataSource()
+    updateMonthsDataSource()
+    scrollToCurrentMonth(animated: false)
+    textField.inputAccessoryView = UIView()
+  }
+
+  /// setup keyboard configuration
+  private func setupKeyboard(with configuration: VGSExpDateConfiguration) {
+    textField.keyboardType = configuration.keyboardType ?? configuration.type.keyboardType
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSExpDateTextField.swift:145]: Would suggest to hardcode number keyboard.

File: Sources/VGSCollectSDK/Core/Enums.swift [hunk 2]
```diff
@@ -42,6 +42,16 @@ public enum FieldType: Int, CaseIterable {
     case ssn
 }
 
+/// Type of `VGSTextField` inout source
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/Core/Enums.swift:45]: Update internal docs, put `.`.

File: Sources/VGSCollectSDK/Core/Enums.swift [hunk 3]
```diff
@@ -42,6 +42,16 @@ public enum FieldType: Int, CaseIterable {
...
+public enum VGSTextFieldInputSource {
+  
+  /// UIKeyboard input type
+  case keyboard
+  
+  /// UIDatePicker input type
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/Core/Enums.swift:51]: Update internal docs, put `.`.

File: Sources/VGSCollectSDK/Core/Enums.swift [hunk 4]
```diff
@@ -42,6 +42,16 @@ public enum FieldType: Int, CaseIterable {
     case ssn
 }
 
+/// Type of `VGSTextField` inout source
+public enum VGSTextFieldInputSource {
+  
+  /// UIKeyboard input type
+  case keyboard
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/Core/Enums.swift:49]: Update internal docs, put `.`.

File: Sources/VGSCollectSDK/Core/VGSExpDateConfiguration.swift [hunk 5]
```diff
@@ -13,7 +13,10 @@ import UIKit
 
 /// A class responsible for configuration `VGSTextField` with `fieldType = .expDate`. Extends `VGSConfiguration` class.
 public final class VGSExpDateConfiguration: VGSConfiguration, VGSFormatSerializableProtocol {
-       
+   
+  /// Input Source type. Default is `VGSTextFieldInputSource.datePicker`
```
Comment (EugeneIOs) [inline Sources/VGSCollectSDK/Core/VGSExpDateConfiguration.swift:17]: Update internal docs, put `.`.

Comment (dmytrokhl) [reply Sources/VGSCollectSDK/Core/VGSExpDateConfiguration.swift]: fixed

File: Sources/VGSCollectSDK/Core/Enums.swift (hunk 4 above)
Comment (dmytrokhl) [reply Sources/VGSCollectSDK/Core/Enums.swift:49]: fixed

File: Sources/VGSCollectSDK/Core/Enums.swift (hunk 3 above)
Comment (dmytrokhl) [reply Sources/VGSCollectSDK/Core/Enums.swift]: fixed

File: Sources/VGSCollectSDK/Core/Enums.swift (hunk 2 above)
Comment (dmytrokhl) [reply Sources/VGSCollectSDK/Core/Enums.swift]: fixed

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSExpDateTextField.swift (hunk 1 above)
Comment (dmytrokhl) [reply Sources/VGSCollectSDK/UIElements/Text Field/VGSExpDateTextField.swift:145]: default keyboard for `.expDate` is decimal. We give some flexibility to choose other keyboard if needed. Format pattern will not allow to type other then digits symbols.

**Review by EugeneIOs — APPROVED** [review]

**Pull request merged**